sys/log/src/log_nmgr.c

```diff
@@ -76,17 +76,30 @@ log_nmgr_add_entry(struct log *log, void *arg, void *dptr, uint16_t len)
     json_encode_object_start(encoder);
 
     JSON_VALUE_STRINGN(&jv, data, rc);
-    rc = json_encode_object_entry(encoder, "log", &jv);
+    rc = json_encode_object_entry(encoder, "msg", &jv);
     if (rc != 0) {
         goto err;
     }
 
-    JSON_VALUE_UINT(&jv, *(uint64_t *) &ueh.ue_ts);
+    JSON_VALUE_INT(&jv, ueh.ue_ts);
     rc = json_encode_object_entry(encoder, "ts", &jv);
     if (rc != 0) {
         goto err;
     }
 
+    JSON_VALUE_UINT(&jv, ueh.ue_level);
+    rc = json_encode_object_entry(encoder, "level", &jv);
+    if (rc != 0) {
+        goto err;
+    }
+
+    /* Dummy index value for now */
+    JSON_VALUE_UINT(&jv, ueh.ue_index);
+    rc = json_encode_object_entry(encoder, "index", &jv);
```

Comment:
It is for the ordering for events that happen in a millisecond. I am changing it to be a byte as per the discussion and suggestion.